test/cases/api.sh

```diff
@@ -512,6 +512,12 @@ function createReqFileAWS() {
 {
   "distribution": "$DISTRO",
   "customizations": {
+    "filesystem": [
+      {
+        "mountpoint": "/var",
+        "min_size": 1024
```

Comment:
I think we don't guard against `/var` being too small, so I think the build will actually fail on this value. We are asking for a 1kB filesystem, which isn't going to work. I think this should be fixed in the filesystem code cc @gicmo @kingsleyzissou

Reply:
I took the liberty of pushing a fix for this. The errors in CI were really not all that helpful/clear, so I thought I'd quickly verify that this indeed is the issue.

Reply:
I think `125MB` might still be a bit too small and that's why the one test is still failing, it might be worthwhile bumping that up. We can then add validation as a follow-up to the luks/lvm PR once that lands.

Reply:
Oops, I was a bit optimistic there.

Reply:
Fixed this.